res/problems/database-read/problem.md

```diff
@@ -0,0 +1,54 @@
+Write a program that receives a database connection string (DSN). Connect to the database, query it and update some data.
+
+Display the information of all the users in the database table `users` who's is are over 30. Print out each row as a new line formatted like:
+
+`User: Jim Morrison Age: 27 Sex: Male` followed by a line break.
+
+Finally you will be given a random name as the second argument to your program, you should update the row in the `users` table 
+which corresponds to this name. You should change the name to `David Attenborough`.
```

Comment:
are these newline chars?  L14/39 too. Better to remove and let the terminal wrap.